swayfmt/tests/mod.rs

```diff
@@ -624,9 +620,9 @@ pub enum Bazz { // Here is a comment
 pub enum Bazz { // Here is a comment
     // Trying some ASCII art
     baz: (),
-    bazzz: (),//-----
-              //--D--
-              //-----
+    bazzz: (), //-----
+    //--D--
+    //-----
```

Comment:
This is unindented because it would be difficult to support this kind of comment given the current heuristics/assumptions, but I'm also not sure if we even want this sort of alignment to begin with? Unless there is some context i am missing here.

Reply:
Yeah I think I'm content with not supporting this kind of aligned trailing-yet-multiline comment - it's pretty niche, and I think we've come to terms with the fact we may need to be at least slightly opinionated on what we allow for comments.